BREAKING.md

```diff
@@ -20,13 +20,20 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 ## 2.0.0-internal.3.4.0 Upcoming changes
 
 [IResolvedUrl will be equivalent to IFluidResolvedUrl](#IResolvedUrl-will-be-equivalent-to-IFluidResolvedUrl)
+[LoaderHeader.cache deprecated]
+(#LoaderHeader.cache-deprecated)
```

Comment:
This doesn't render correctly because of the line break (see https://github.com/microsoft/FluidFramework/blob/fa3ff191fd38ec51119b5fdc3d93c44587944209/BREAKING.md)

Reply:
The anchor link should also be `#loaderheadercache-deprecated` 

See https://github.com/microsoft/FluidFramework/blob/fa3ff191fd38ec51119b5fdc3d93c44587944209/BREAKING.md#loaderheadercache-deprecated